lib/matplotlib/tests/test_colors.py

```diff
@@ -18,6 +18,7 @@
 import matplotlib.pyplot as plt
 import matplotlib.scale as mscale
 from matplotlib.testing.decorators import image_comparison, check_figures_equal
+from re import escape
```

Comment:
Since `re` is part of the python standard library it should be imported at the top with `copy`, etc.